### PR TITLE
AmazonSQS runtime fixes related to region selection

### PR DESCRIPTION
**Pull request by ramonsmits**

### CODE REVIEW

File: src/PerformanceTests/Transport.V7.AmazonSQS.V4/AmazonSQSProfile.cs [hunk 1]
```diff
@@ -20,14 +22,26 @@ public void Configure(EndpointConfiguration endpointConfiguration)
         // https://docs.particular.net/transports/sqs/configuration-options
         var transport = endpointConfiguration.UseTransport<SqsTransport>();
 
+        NServiceBus.Logging.LogManager.GetLogger(nameof(AmazonSQSProfile)).Info(RegionEndpoint.GetBySystemName(cs["Region"].ToString()).DisplayName);
+
+        var credentials = new EnvironmentVariablesAWSCredentials();
+        var region = RegionEndpoint.GetBySystemName(cs["Region"].ToString());
```
Comment (danielmarbach) [inline src/PerformanceTests/Transport.V7.AmazonSQS.V4/AmazonSQSProfile.cs]: Why not first extract the region by name, then log the display name? So essentially refactor it to following:

```
var region = RegionEndpoint.GetBySystemName(cs["Region"]);
LogManager.GetLogger(nameof(AmazonSQSProfile)).Info(region.DisplayName);
// rest
```

File: src/PerformanceTests/Transport.V7.AmazonSQS.V4/AmazonSQSProfile.cs [hunk 2]
```diff
@@ -20,14 +22,26 @@ public void Configure(EndpointConfiguration endpointConfiguration)
         // https://docs.particular.net/transports/sqs/configuration-options
         var transport = endpointConfiguration.UseTransport<SqsTransport>();
 
+        NServiceBus.Logging.LogManager.GetLogger(nameof(AmazonSQSProfile)).Info(RegionEndpoint.GetBySystemName(cs["Region"].ToString()).DisplayName);
```
Comment (danielmarbach) [inline src/PerformanceTests/Transport.V7.AmazonSQS.V4/AmazonSQSProfile.cs]: Formatting

File: src/PerformanceTests/Transport.V7.AmazonSQS.V4/Transport.V7.AmazonSQS_v4.csproj [hunk 3]
```diff
@@ -11,7 +11,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="7.0.0-beta0009" />
```
Comment (danielmarbach) [inline src/PerformanceTests/Transport.V7.AmazonSQS.V4/Transport.V7.AmazonSQS_v4.csproj]: 👍

File: src/PerformanceTests/Transport.V7.AmazonSQS.V4/AmazonSQSProfile.cs (hunk 2 above)
Comment (ramonsmits) [reply src/PerformanceTests/Transport.V7.AmazonSQS.V4/AmazonSQSProfile.cs]: This was actually just for testing, I'll remove it.

**Pull request merged**